modules/rosapps/include/CMakeLists.txt

```diff
@@ -1 +0,0 @@
-add_subdirectory(vfd)
```

Comment:
You stripped the file `modules/rosapps/include/vfd/CMakeLists.txt` . How does configure and ninja know now that they have to build the module?

Reply:
I see no module here.

Reply:
Last question: Is the vfd.sys still copied into the resulting iso? if you do
```
configure -DENABLE_ROSAPPS=1
ninja bootcd
```

Reply:
Yes, as can be checked on this PR build artifacts.